components/activity-section.jsx

```diff
@@ -13,25 +13,33 @@ var ActivitySection = React.createClass({
           Section {this.props.title}
         </h2>
         {this.props.activities.map(function (activity, i) {
-          return (
-            <div className="activity" key={i}>
-              <Illustration height={122} width={122}
-              src1x={activity.image1x}
-              src2x={activity.image2x}
-              alt=""
-              link={activity.link}>
-                <div className="curriculum-description">
-                  <a href={activity.link}>
-                    <h3>{activity.title}</h3>
-                  </a>
-                  <p>
-                    <em>{activity.subtitle}</em>
-                  </p>
-                  <p className="description" dangerouslySetInnerHTML={{__html: activity.description}}></p>
-                </div>
-              </Illustration>
-            </div>
-          )
+          if (Object.keys(activity).length === 1 && activity.title) {
```

Comment:
Hmm, for these situations, what do you think of changing the `title` property to `placeholderText` or something a bit more semantically meaningful? That could make reading the code a bit more self-explanatory, and also avoid the `Object.keys` length check.